sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -0,0 +1,987 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Administration;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Tests
+{
+    public class AnomalyAlertConfigurationLiveTests : MetricsAdvisorLiveTestBase
+    {
+        public AnomalyAlertConfigurationLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [RecordedTest]
+        public async Task CreateAndGetAlertConfigurationWithWholeSeriesScope()
+        {
+            MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient();
+
+            var scope = MetricAnomalyAlertScope.GetScopeForWholeSeries();
+            var metricAlertConfig = new MetricAnomalyAlertConfiguration(DetectionConfigurationId, scope);
+
+            string configName = Recording.GenerateAlphaNumericId("config");
+            var metricAlertConfigs = new List<MetricAnomalyAlertConfiguration>() { metricAlertConfig };
+
+            var configToCreate = new AnomalyAlertConfiguration(configName, new List<string>(), metricAlertConfigs);
+
+            await using var disposableConfig = await DisposableAlertConfiguration.CreateAlertConfigurationAsync(adminClient, configToCreate);
+
+            AnomalyAlertConfiguration createdConfig = await adminClient.GetAlertConfigurationAsync(disposableConfig.Id);
+
+            Assert.That(createdConfig.Id, Is.EqualTo(disposableConfig.Id));
+            Assert.That(createdConfig.Name, Is.EqualTo(configName));
+            Assert.That(createdConfig.Description, Is.Empty);
+            Assert.That(createdConfig.CrossMetricsOperator, Is.Null);
+            Assert.That(createdConfig.IdsOfHooksToAlert, Is.Not.Null.And.Empty);
```

Comment:
what is the difference between having null and empty here? looks like the service will still succeed on creating a alertConfiguration even if no hooks are added.

Reply:
It should never be null. If no hooks are added, it's expected to be empty. You can see that I've added a `Is.Not.Null` check in every test, regardless of hooks being added or not.

Reply:
Sorry, I commented in the wrong spot.
I mean for the input parameter. If the service allows no hooks, what is the difference between the user sending null and an empty list?

Reply:
User can't send null. Currently they'll get an `ArgumentNullException` if they try to pass a null value to the `AnomalyAlertConfiguration` constructor.

Once [this design issue](https://github.com/Azure/azure-sdk-for-net/issues/16321) gets fixed, the empty list of hooks will be available (and not settable) by default, without being passed to the constructor. Possibly something like:
```cs
var config = new AnomalyAlertConfiguration();
config.IdsOfHooksToAlert.Add(hook);
```